mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -200,6 +200,7 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_6085 setLivingFlag (IZ)V
 		ARG 1 mask
 		ARG 2 value
+	METHOD method_6086 acceptsThrownPotions ()Z
```

Comment:
"accepts" is a weird term for this. Why not `affectedBySplashPotions`?

Reply:
I like that suggestion., though I would debate between that and isAffectedBySplashPotions

Reply:
`doSplashPotionsHaveAnyEffectsWhenAppliedToThisEntity()`?

Reply:
True, `affectedBySplashPotions` would be a field name. It could be argued that `isAffectedBySplashPotions` is a tad too verbose, as Sollace noted, but at least it's clear.

Reply:
I'm a bit cautious to refer to them as "splash potions" since it also affects the behavior of lingering potions, but I suppose you could consider them a type of splash potion.